pages/linux/pacman-database.md

```diff
@@ -0,0 +1,28 @@
+# pacman --database
+
+> Arch Linux package manager utility.
```

Comment:
Could this be more descriptive of the "subcommand" being talked about here?

Reply:
```suggestion
> Operate on the Arch Linux package database.
> Modify certain attributes of the installed packages.
```

Reply:
Oh! I was going to suggest something like this for all of the `pacman --xxx` pages.